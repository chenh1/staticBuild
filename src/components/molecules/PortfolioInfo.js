import React, {PropTypes} from 'react';
import { SubHeader, Paragraph } from '../atoms';

const PortfolioInfo = ({subHeaderText, paragraphText}) => {
    return (
        <div className="info">
            <SubHeader text={subHeaderText}/>
            <Paragraph text={paragraphText}/>
        </div>
    );
};

PortfolioInfo.propTypes = {
    subHeaderText: PropTypes.string.isRequired,
    paragraphText: PropTypes.array.isRequired
};

export default PortfolioInfo;