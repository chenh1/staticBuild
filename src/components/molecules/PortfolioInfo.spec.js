import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import PortfolioInfo from './PortfolioInfo';

test('renders container for subheader and paragraph', () => {
    const tree = renderer.create(<PortfolioInfo
        subHeaderText="Mock sub-header text"
        paragraphText={["Mock paragraph text"]} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});