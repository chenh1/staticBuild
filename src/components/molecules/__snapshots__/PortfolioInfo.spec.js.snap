// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders container for subheader and paragraph 1`] = `
<div
  className="info"
>
  <h3
    className="header-three"
  >
    Mock sub-header text
  </h3>
  <p
    className="paragraph-one"
  >
    Mock paragraph text
  </p>
</div>
`;
